tests/test_spherePoint.py

```diff
@@ -135,6 +135,10 @@ def testDoubleDoubleUnitsConstructor(self):
         self.assertEqual(sp2, SpherePoint(1.1 * radians, -0.6 * radians))
         with self.assertRaises(TypeError):
             SpherePoint(-42.0, 45.0)  # units must be specified
+        with self.assertRaises(pexEx.InvalidParameterError):
+            SpherePoint(-42.0, 91.0, degrees)  # latitude out of range
+        with self.assertRaises(pexEx.InvalidParameterError):
+            SpherePoint(-42.0, -91.0, degrees)  # latitude out of range
```

Comment:
Could you also add a test that e.g. negative longitudes get wrapped correctly? (Possibly as part of `testGetLongitudeValue` rather than here, since that's where the tests are done for the original constructor).

Reply:
I don't know if you want the other tests from `testInit2ArgErrors`, like infinite latitude.